stapi-fastapi/tests/conftest.py

```diff
@@ -1,9 +1,15 @@
+import sys
 from collections.abc import AsyncIterator, Callable, Generator, Iterator
 from contextlib import asynccontextmanager
-from datetime import UTC, datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from typing import Any
 from urllib.parse import urljoin
 
+if sys.version_info >= (3, 11):
+    from datetime import UTC
+else:
+    UTC = timezone.utc
```

Comment:
Rather than this, it might be simpler to just update all `UTC` references to `datetime.timezone.utc`? That way we don't have to change anything when we drop 3.10 support.